apps/condo/bin/lib/billing-context-script-core.js

```diff
@@ -0,0 +1,164 @@
+const dayjs = require('dayjs')
+const path = require('path')
+const { get, map, isEmpty } = require('lodash')
+
+const { BillingIntegrationOrganizationContext } = require('@condo/domains/billing/utils/serverSchema')
+const { CONTEXT_FINISHED_STATUS } = require('@condo/domains/miniapp/constants')
+
+const { SendPushScriptCore, runFnWithArgs } = require('./send-push-script-core')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+const { COUNTRIES, DEFAULT_LOCALE } = require('@condo/domains/common/constants/countries')
+
+const EXEC_COMMAND = 'yarn workspace @app/condo node'
+const BASE_NAME = path.posix.basename(process.argv[1])
+const MAX_SEND_COUNT = 100
+const DATE_FORMAT = 'YYYY-MM-DD'
+const FORCE_SEND = 'FORCE_SEND'
+
+class BillingContextScriptCore extends SendPushScriptCore {
```

Comment:
1) Try to avoid a long chain of inheritance. You can create just a function and you don't need to create a class.

move everything related to utils inside one file. Don't split it into three files.

Reply:
If you solve just one task it's better to move everything inside one file rather than create three different files with different abstractions levels. Or you need to migrate another files to use your abstractions.

Reply:
reduced number of inheritance chains :)